documentation/publish/distribute-for-desktop-apps.md

```diff
@@ -27,17 +27,27 @@ date: 2019-07-30 02:28:37
 
 {% capture content %}
 
-If you have developed an add-on to complement a desktop application, there are two ways you can install the add-on:
+<!-- Alert -->
 
-- Direct the user to install the add-on from the web, either from [addons.mozilla.org (AMO)](https://addons.mozilla.org) or your own site.
-- Sideloading via standard extension folders or the Windows registry.
+{% capture alert %}
 
-Of these options, directing the user to install from AMO is recommended. The reasons for recommending this option are:
+Starting with Firefox 74, it is no longer be possible to have an extension be automatically installed as part of another application install. See the [Add-ons Blog](https://blog.mozilla.org/addons/2020/03/10/support-for-extension-sideloading-has-ended/) for more information.
 
-- It avoids any issues with the installation process; the user will not get an interstitial messages during the installation of the add-on, find the add-on installed but disabled, or find that the add-on was not installed.
-- If you update the add-on, the new version will be automatically installed.
+{% endcapture %}
+{% include modules/note.html
+	content=alert
+	alert=true
+%}
+
+<!-- END: Alert -->
+
+If you have developed an add-on to complement a desktop application, you will need to provide a way for users to install your extension. You can direct the user to install the add-on from your website or list it on addons.mozilla.org (AMO).
+
+Of these options, directing the user to install from AMO is recommended. It avoids any issues with the installation process; the user will not get an interstitial messages during the installation of the add-on, find the add-on installed but disabled, or find that the add-on was not installed.
+
+If you would like to direct users to install the add-on from your website, please read this article on [self-distribution](https://extensionworkshop.com/documentation/publish/submitting-an-add-on/#self-distribution) to learn how to prepare the .xpi file for web installs.
```

Comment:
Please do not link to complete website link, connect the page internally see in other pages please how it connects